src/index.js

```diff
@@ -1,17 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App.js';
-import reportWebVitals from './reportWebVitals.js';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
```

Comment:
- [ ] the `App` component should call the `Calculator` component and it should be class-based as well.

Reply:
Resolved